lib/binding_web/binding.js

```diff
@@ -842,78 +842,90 @@ class Language {
           const operator = steps[0].value;
           let isPositive = true;
           let matchAll = true;
+          let captureName;
           switch (operator) {
             case 'any-not-eq?':
-              isPositive = false;
```

Comment:
The `switch` statement fallthrough trick fails to work when there's a 2x2 matrix of flags — there's no way to structure them so that they don't get in each other's way. Instead, I check for the presence of `any-` at the start of `operator`.